scripts/2.mutation-classifier.py

```diff
@@ -326,19 +326,45 @@ def get_threshold_metrics(y_true, y_pred):
 pd.crosstab(coef_df.feature_set, np.sign(coef_df.weight).rename('coefficient_sign'))
 
 
+# ### Top coefficients for covariates model
+
 # In[21]:
 
-# Top standardized coefficients
-(coef_df
-    .query("feature_set == 'full'")
-    .head(10)
-)
+coef_df.query("feature_set == 'covariates'").head(10)
 
 
-# ## Investigate the predictions
+# ### Top coefficients for full model
 
 # In[22]:
 
+coef_df.query("feature_set == 'full'").head(10)
```

Comment:
> If we were to try and compare gene coefficients between the two models it would be difficult to determine what effect including covariates had on coefficients because the addition of coefficients isn't the only thing that changed

Good point. Given this consideration my opinion is:

1. In the case of gene-level coefficients, a combined dataframe is most useful.
2. In the case of component-level coefficients, a combined dataframe is less essential but still likely preferable. Of course, you can note that different models may have received different numbers of components. And some coefficient values will be missing for some models (since those components were not evaluated by the logistic regression).

However, ease of implementation is also important. I don't want to burden the notebook with lot's of code to get these consolidated coefficient dfs. But I do think they will be most useful.